script/Ciel/Day 5/05_01_CIEL05_3.txt

```diff
@@ -473,7 +474,7 @@ Do you get it?
 -- Page 422, Offset 11994.
 -- あの子は特別だから。きっちり対策をして装備を調えていたらいい勝負になってたんじゃない？　さっきは下調べのつもりだったから簡易装備しかなかったけど」
 That girl is special.
- Don't you think she'd be on even footing with him if she had a plan and the necessary equipment?
+ I guess she'd be on even footing with him if she had a plan and the necessary equipment.
```

Comment:
This doesn't sit right with the rising/questioning intonation to me, maybe
> There's a chance she might actually be on even footing with him if she had a plan and the necessary equipment?

Reply:
I prefer the gpr version over that, even with a questioning intonation, that just feels like a little much imo.